pkg/interface/src/views/apps/chat/ChatResource.tsx

```diff
@@ -89,7 +89,7 @@ function ChatResource(props: ChatResourceProps) {
         props.association.resource,
         msg.index
       );
-      return `${url}\n~${msg.author} : `;
+      return `${url}\n`;
```

Comment:
we aren't mentioning people in replies anymore?

Reply:
cc: @urcades

Reply:
Yes, we are removing them — I wanted to make sure though, will people still receive notifications whenever their messages are replied to or referenced?

Reply:
To be clear:

If notifications (for people using your posts as references) are reliant on **explicit** mentions, we need to generally engineer these references such that notifications are generated whenever one of your nodes are referenced.

If this isn't easy for whatever reason, or out of scope, we should continue to include them in the reply text

cc @matildepark — let me know if this ask (generating hark notifications whenever one's node is referenced) is out of scope for this work

Reply:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/20846414/117221983-62610a00-add8-11eb-86a9-0661d64e4a7f.png">

This did not notify me

Reply:
That is definitely 100% out of scope of this PR @urcades, let's just retain mentions

Reply:
damn it, ok — will cover the mechanics of this UX in another issue